Documentation added tutorials-evaluating-script-structure.rst

magdesign opened the pull request:
added on request in the [forum](https://discuss.pixls.us/t/call-for-volunteers-merge-http-opticalenquiry-com-nuke-into-natron-doc/24690) the section:
http://www.opticalenquiry.com/nuke/index.php?title=Evaluating_Script_Structure
from Martin Constable to the docs.
new file is: tutorials-evaluating-script-structure.rst
- added 3 new images
- added some  link targets for cross referencing
- added entry to show in toctree

Commit:
Documentation: Evaluating Script Structure

added on request

Commit:
added index entry

Commit:
Documentation: Composing Channels, Composing Colors

devernay commented:
Thank you!
I'll check this when I have time, unless someone else wants to review.
The build is available at: https://natron--612.org.readthedocs.build/en/612/

devernay commented on Documentation/source/guide/compositing-color.rst:
```diff
@@ -1,8 +1,103 @@
+
+
+
+**Add****
```
too many `*`

devernay commented on Documentation/source/guide/compositing-color.rst:
```diff
@@ -1,8 +1,103 @@
+Positive values will brighten all parts of the image. As a stand alone color manipulation it is of limited use, though it is ideal for lightening the blacks of distant objects.
+
+
+**Multiply***
```
too many `*`

devernay commented on Documentation/source/guide/compositing-color.rst:
```diff
@@ -1,8 +1,103 @@
+Try also blurring the a and b channels. This will blur only the hue and saturation components of an image and leave its lightness values alone.
+A novel use for the ColorSpace node can be found in the Assets page (see the Double Rainbows asset).
+
+**ColorCorrect** *
```
dangling `*`

devernay commented on Documentation/source/guide/compositing-color.rst:
```diff
@@ -1,8 +1,103 @@
+
+This raises or lowers the middle-ish point of the color curve. The default value is one, with smaller numbers darkening the lower registers and higher numbers lightening them. There is a Gamma node but I find that stand-alone gamma adjustments are best done using ColorLookup so as to give you flexibility over where the ‘grab point‘ of your curve is. Both ColorCorrect and Grade have built in Gamma ...
+
+**Grade** *
```
`*`

devernay commented on Documentation/source/guide/compositing-color.rst:
```diff
@@ -1,8 +1,103 @@
+
+**Histogram**
+
+**HueCorrect** *
```
`*`

devernay commented on Documentation/source/guide/compositing-color.rst:
```diff
@@ -1,8 +1,103 @@
+When adjusting color it is important to have separate control over these values, which is something that the HueCorrect node offers.
+This effect it masks by two further values: hue and saturation. Its interface offers control over nine values:
+
+Saturation (sat)
```
format as title?

devernay commented on Documentation/source/guide/tutorials-evaluating-script-structure.rst:
```diff
@@ -0,0 +1,208 @@
+   .. _evaluating-script-structure:
+
+
+Thanks to Martin Constable to allow us to use his documents.
```
Would add Martin (and you) to the Documentation authors (https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTORS.txt and in the intro)

Commit:
fixed the * issues, added name to contributors.txt

magdesign commented:
- Fixed the issues with asterisk * signs
- Added name to CONTRIBUTORS.txt
- "working with color" was always there as title in prev. versions, left it like this
